CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+## How to contribute
+
+You are encouraged to contribute to the repository by **forking and submitting a pull request**.
+
+For significant changes, please open an issue first to discuss the proposed changes to avoid re-work.
```

Comment:
If I'm not mistaken we have a similar sentence below. I would suggest to replace this one with the second one and remove the second one. So, I mean put here:

"If you would like **to propose a significant change, please open an issue first** to discuss the work with the community and avoid re-work."